main.go

```diff
@@ -32,6 +34,19 @@ var (
 )
 
 func init() {
+	// Change glog default log level to INFO.
+	// Note glog is not initialized yet, so we can only use fmt for printing
+	// errors.
+	err := flag.Set("logtostderr", "true")
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
+	}
+	err = flag.Set("stderrthreshold", "INFO")
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
+	}
```

Comment:
If we're disabling file logs and writing everything to stderr, I wonder if we should just move away from glog entirely. 🤔 

Separately, this doesn't actually change the default value - it changes the flag value before it's parsed. Functionally it does the right thing, but this may disrupt documentation tools that print out the default flag value.

Reply:
Agree to move away from glog entirely, do you have a default logging library you would choose instead?
I think moving away from glog is ideally a separate PR, because it's not very related here.

I'd propose to keep this as is, because it's an improvement over default values of glog. And we add a TODO item in v2 ROADMAP to replace glog. What do you think?

Reply:
Added to roadmap: https://github.com/google/go-licenses/issues/70

Reply:
The stdlib log package is probably fine? But punting this to another PR SGTM.